### ISSUE TITLE
Focus Lock does not allow navigating into search results after search all is completed

### ISSUE THREAD

**Issue by strangerinusall** [pjeby/pane-relief#59]
Hi, when Focus Lock is disabled it is possible to navigate into search result from search field via keyboard (down arrow), but since last update it does not work. So Search All -> Arrow Down to select result does not work.

<img width="294" alt="image" src="https://github.com/user-attachments/assets/2af777b1-f403-47de-8c09-a847fbbb0827">

**Comment by pjeby** [comment]
Sorry, are you saying it does this when focus lock is *disabled*?  Note that focus lock has never allowed you to navigate the search results with the keyboard before, AFAIK.

**Comment by strangerinusall** [comment]
When Focus Lock is enabled. And yes, I know (I am the same guy who commented on quick-explorer topic).

In the latest update you made so that it's possible to activate search while Focus Lock is enabled. I am simply saying that the way search works now is slightly worse than search without Focus Lock, since ideally you might want to use the keyboard to jump from search field into search results. Otherwise you have to use mouse to look over search results.

**Comment by pjeby** [comment]
Yes, I'm just noting that focus lock always worked that way - the entire point of focus lock is that it doesn't allow sidebar panes to become active, and especially not to get keyboard focus.  The search box working at all was originally a happy accident, which went away in recent Obsidian versions.  Really, the reveal active file plugin should be fixed to not have this issue in the first place, as most of the other reasons for Focus Lock existing have gone away (apart from popout window behavior),

**Comment by strangerinusall** [comment]
Ah now I understand! I will probably create an issue request in Reveal Active File page, and this can be then closed.

**Comment by pjeby** [comment]
Obsidian 1.8 now has an option to auto-reveal the active file (via a button at the top of the explorer), so focus lock's only remaining use is to keep the main window from stealing focus from popouts.  So I'm going to go ahead and close this now.
